src/pages/accommodations/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Add } from '@mui/icons-material';
+import { Box, IconButton, Skeleton, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+import { useGetAccommodations } from '@/api/queries/accommodations/useGetAccommodations';
+import { ROUTES } from '@/config/routes.config';
+import { lineClampStyle } from '@/utils';
+
+export default function Accommodations() {
+  const { data: accommodations, isLoading } = useGetAccommodations();
+
+  const onError = (e: React.SyntheticEvent<HTMLImageElement, Event>) => {
+    e.currentTarget.src = 'https://via.placeholder.com/300';
+  };
+
+  return (
+    <Box>
+      <Box display="flex" alignItems="center" justifyContent="space-between" my={8}>
+        <h1>Your Accommodations</h1>
+        <Link to={ROUTES.accommodations.create}>
+          <IconButton>
+            <Add />
+          </IconButton>
+        </Link>
+      </Box>
+      {/* List */}
+      <Box display="grid" gap={8} gridTemplateColumns={'repeat(auto-fill, minmax(280px, 1fr))'}>
+        {isLoading
+          ? Array.from({ length: 10 }).map((_, i) => (
```

Comment:
create component for the Skeleton view. looks like it useful for many components